<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaborFilter</class>
 <widget class="QDialog" name="GaborFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GaborFilter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="InputMaskFactorLable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>InputMaskFactor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="InputMaskLable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>InputMask</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="InputMask">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSlider" name="InputMaskFactorHorizontalSlider">
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="psiLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>psi</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QDoubleSpinBox" name="LambdaDoubleSpinBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="InputIMG">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="InputIMGLable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>InputIMG</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QDoubleSpinBox" name="psiDoubleSpinBox">
     <property name="maximum">
      <double>360.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="LambdaLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Lambda</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="KernelSizeHeightLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>KernelSizeHeight</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QSpinBox" name="KernelSizeHeightSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLabel" name="GammaLabel">
     <property name="text">
      <string>Gamma</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="SigmaLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sigma</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="GaborIMG">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="GaborIMGLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>GaborIMG</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QDoubleSpinBox" name="ThetaDoubleSpinBox">
     <property name="maximum">
      <double>360.990000000000009</double>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QDoubleSpinBox" name="GammaDoubleSpinBox"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QSpinBox" name="InputMaskFactorSpinBox">
     <property name="maximum">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="ThetaLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Theta</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="GaborIMGCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>GaborIMG surce</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QDoubleSpinBox" name="SigmaDoubleSpinBox"/>
   </item>
   <item row="13" column="1">
    <widget class="QSpinBox" name="KernelSizeWidthSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="KernelSizeWidthLable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Kernel Size Width</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
